gremlin/templates/secret.yaml

```diff
@@ -12,8 +12,8 @@ metadata:
     version: v1
 type: kubernetes.io/Opaque
 stringData:
-  GREMLIN_CLUSTER_ID: {{ default .Values.gremlin.clusterID .Values.gremlin.secret.clusterID | required "required: .Values.gremlin.secret.clusterID" | toString }}
-  GREMLIN_TEAM_ID: {{ default .Values.gremlin.teamID .Values.gremlin.secret.teamID | required "required: .Values.gremlin.secret.teamID" | toString }}
+  GREMLIN_CLUSTER_ID: {{ (default .Values.gremlin.clusterID .Values.gremlin.secret.clusterID | required "required: .Values.gremlin.secret.clusterID") | toString }}
+  GREMLIN_TEAM_ID: {{ (default .Values.gremlin.teamID .Values.gremlin.secret.teamID | required "required: .Values.gremlin.secret.teamID") | toString }}
 {{- if (eq (include "gremlin.secretType" .) "secret") }}
   GREMLIN_TEAM_SECRET: {{ .Values.gremlin.secret.teamSecret | required "required: .Values.gremlin.secret.teamSecret" |  toString }}
```

Comment:
While unlikely, `GREMLIN_TEAM_SECRET` presumably can be subject to this just as much as `GREMLIN_TEAM_ID`. Let's make the change here too.

Reply:
Updated to include team secret, added patch versions to integration/regular chart.yaml and included integration chat teamid and secret